CRM/Batch/Form/Entry.php

```diff
@@ -937,9 +937,6 @@ protected function emailReceipt($form, &$formValues): bool {
     $form->assign('module', 'Membership');
     $form->assign('contactID', $formValues['contact_id']);
 
-    $form->assign('membershipID', CRM_Utils_Array::value('membership_id', $form->_params, CRM_Utils_Array::value('membership_id', $form->_defaultValues)));
-    $this->assign('contributionID', $this->getCurrentRowContributionID());
```

Comment:
Just double-checking: if someone customized their receipt templates to use `membershipID` and `contributionID`, now they should update to use `membershipId` / `contributionId`? If yes, could we support both? (hard to guess smarty variables often make devs cry)

Reply:
@mlutfy no the smarty variables are `$contributionID` and `$membershipID`